backend/src/StdLibExecution/Libs/List.fs

```diff
@@ -354,7 +354,7 @@ let fns : List<BuiltInFn> =
           uply {
             let f (dv : Dval) : Ply<bool> =
               uply {
-                let! result = Interpreter.applyFnVal state fn [ dv ]
+                let! result = Interpreter.applyFnVal state 0UL fn [] [ dv ]
```

Comment:
wondering - are there any negative consequences of using `0UL` for all of these internal Interpreter calls?

Reply:
It means traces will give poor results - it won't know where to put them.